lib/wordAnalysis.js

```diff
@@ -0,0 +1,43 @@
+
+function wordStats(arrayOfWords) {
+  let counts = {};
+  for (let word of arrayOfWords) {
+    if (!counts[word]) {
+      counts[word] = 1;
+    } else {
+      counts[word] += 1;
+    }
+  }
+  let greatestFrequencies = Object.values(counts).sort().reverse().slice(0, 5);
```

Comment:
`Object.entries` gives you back an array of key/value pairs (as two-element arrays). You can do:

```javascript
let frequentWords = Object.entries.sort(function(leftPair, rightPair) {
  return leftPair[1] > rightPair[1];
}).slice(0, 5);
```

Print that out and you'll see what you get.